python/cudf/setup.py

```diff
@@ -16,7 +16,20 @@
 
 CUDA_HOME = os.environ.get("CUDA_HOME", False)
 if not CUDA_HOME:
-    CUDA_HOME = os.path.dirname(os.path.dirname(shutil.which("cuda-gdb")))
+    path_to_cuda_gdb = shutil.which("cuda-gdb")
+    if path_to_cuda_gdb is None:
+        raise OSError(
+            "Could not locate CUDA. "
+            "Please set the environment variable "
+            "CUDA_HOME to the path to the CUDA installation "
```

Comment:
Maybe "or add `cuda-gdb` to the `PATH`"?

Reply:
IMO setting `CUDA_HOME` is much less intrusive to the user than asking them to modify `PATH`

Reply:
Yeah I figured it would be good to provide both options. Also it includes a clue as to what is failing (namely we can't find `cuda-gdb`). Though no strong feelings if you would prefer to leave the wording as-is.

Reply:
Well, the current approach is by no means bullet-proof. For example, I could have some custom `cuda-gdb` in my `$HOME/bin` directory, but CUDA installed in `/usr/local/cuda`..

I think let's go with this for now and improve as bugs are reported.